packages/cli/lib/projects.js

```diff
@@ -271,6 +271,7 @@ const makeGetTaskStatus = taskType => {
     const spinnies = new Spinnies({
       succeedColor: 'white',
       failColor: 'white',
+      failPrefix: '\033[1m!\033[0m',
```

Comment:
Nice! In the CLI, we've use `chalk` a lot for formatting text output. I suspect what you have here will work just fine across different terminals, but it might be worth confirming that

Reply:
Yeah +1 to using chalk here if possible, just for consistency.